packages/core/stories/form-field/form-field.stories.tsx

```diff
@@ -511,3 +515,281 @@ export const WithInputWithAdornments: ComponentStory<typeof FormField> = (
+  props
+) => {
+  return (
+    <FormFieldGroupContext.Provider
+      value={{
+        variant: "primary",
+        labelPlacement: "right"
+      }}
+    >
+      <StackLayout role={"group"}>
+        <FormField {...props}>
+          <FormLabel>Form Field label left</FormLabel>
+          <Input defaultValue="Value" />
+          <FormHelperText>Helper text</FormHelperText>
+        </FormField>
+        <FormField {...props}>
+          <FormLabel>
+            Form Field label that's extra long. Showing that labels wrap around
+            to the line.
+          </FormLabel>
+          <Input defaultValue="Primary Input value" />
+        </FormField>
+        <FormField {...props}>
+          <FormLabel>Form Field label</FormLabel>
+          <Input defaultValue="Value" />
+          <FormHelperText>Helper text</FormHelperText>
+        </FormField>
+        <FormField {...props}>
+          <FormLabel>Form Field label</FormLabel>
+          <Input defaultValue="Primary Input value" />
+        </FormField>
+      </StackLayout>
+    </FormFieldGroupContext.Provider>
+  );
+};
+
+export const GroupedWithVariant: ComponentStory<typeof FormField> = (props) => {
+  return (
+    <FormFieldGroupContext.Provider
+      value={{
+        variant: "secondary",
+        labelPlacement: "right"
+      }}
+    >
+      <StackLayout>
+        <FormField {...props}>
+          <FormLabel>Form Field label left</FormLabel>
+          <Input defaultValue="Value" />
+          <FormHelperText>Helper text</FormHelperText>
+        </FormField>
+        <FormField {...props}>
+          <FormLabel>
+            Form Field label that's extra long. Showing that labels wrap around
+            to the line.
+          </FormLabel>
+          <Input defaultValue="Value" />
+        </FormField>
+        <FormField {...props}>
+          <FormLabel>Form Field label</FormLabel>
+          <Checkbox defaultValue="Value" />
+        </FormField>
+        <FormField {...props}>
+          <FormLabel>Form Field label</FormLabel>
+          <Input defaultValue="Value" />
+          <FormHelperText>Helper text</FormHelperText>
+        </FormField>
+        <FormField {...props}>
+          <FormLabel>Form Field label</FormLabel>
+          <Input defaultValue="Value" />
+        </FormField>
+      </StackLayout>
+    </FormFieldGroupContext.Provider>
+  );
+};
+
+export const MultiColumnLayout: ComponentStory<typeof FormField> = (props) => {
+  return (
+    <FormFieldGroupContext.Provider
+      value={{
+        variant: "primary",
+        labelPlacement: "right"
+      }}
+    >
+    <StackLayout style={{"--formFieldLabel-width": "100px"} as React.CSSProperties}>
+        <FormField {...props}>
+          <FormLabel>Form Field label left</FormLabel>
+          <Input defaultValue="Value" />
+          <FormHelperText>Helper text</FormHelperText>
+        </FormField>
+        <StackLayout direction={"row"}>
```

Comment:
Whenever you are adding directions props, you could swap them for `direction={{xs: "column", sm: "row"}}` so they don't squish on resize, and your examples become responsive

Reply:
fab! thank you